samples/templates/default-azuredeploy-docker-sql.json

```diff
@@ -74,7 +74,7 @@
             "type": "bool",
             "defaultValue": true,
             "metadata": {
-                "description": "Deploy Application Insights for the FHIR server. Disabled for Microsoft Azure Government (MAG)"
```

Comment:
Shouldn't we still have docker deployments disabled in gov since the linux server isn't available?

Reply:
This message was regarding AppInsights not being available in MAG, which it now is